photon-lib/src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -332,20 +336,32 @@ public boolean isConnected() {
     }
 
     /**
-     * Set the Apriltag Field Layout used by all PhotonVision coprocessors that are (or might later)
-     * connect to this robot. The topic is marked as persistant, so even if you only call this once
-     * ever, it will be saved on the RoboRIO and pushed out to all NT clients when code reboots.
+     * Get the last set {@link AprilTagFieldLayout}. The tag layout is shared between all PhotonVision
+     * coprocessors on the same NT instance-- this method returns the most recent layout set. If no
+     * layout has been set by the user, this is equal to {@link AprilTagFields#kDefaultField}.
+     *
+     * @return The last set layout
+     */
+    public AprilTagFieldLayout getAprilTagFieldLayout() {
```

Comment:
This is kinda weird... I'm not sure of a better way to do this since we need the ATFL to calculate the multitag result. This does not check what layout the coprocessor has, just the last one set from user code. All coprocessors on a NT instance share the same tag layout (and updating the layout through one camera should mean the others also can get that updated layout), but this uses a static field and technically assumes all coprocessors are on the same NT instance.

It could exist in `PhotonCameraSim` instead, but that would require code duplication specifically for sim when setting a tag layout over NT.

The field could also be accessed directly and the getter method removed, but that would require `PhotonCameraSim` to be in the same package as `PhotonCamera`.